src/ecmascript_simd.js

```diff
@@ -120,6 +120,12 @@ function simdCreate(type) {
   return type.fn.apply(type.fn, lanes);
 }
 
+function simdCreateFromBuffer(type, buffer) {
+  var newValue = type.fn.apply(type.fn, lanes);
```

Comment:
Good point.  I needed an array to prevent an exception though.  It doesn't matter that lanes holds 'unknown' values, since the .s_ is overwritten anyways.